MAUI/ImageEditor/migration.md

```diff
@@ -1,6 +1,6 @@
 ---
 layout: post
-title: Migrate from Xamarin to .NET MAUI ImageEditor (SfImageEditor) | Syncfusion
+title: Migrate from Xamarin to .NET MAUI Image Editor (SfImageEditor) | Syncfusion

```

Comment:
Title: Migrate from Xamarin to .NET MAUI Image Editor (SfImageEditor) | Syncfusion